app/Event/ProductionControllerProvider.php

```diff
@@ -48,16 +42,52 @@ function (Application $app) {
             }
         );
 
+        $app[CreateProductionRequestHandler::class] = $app->share(
+            fn (Application $app) => new CreateProductionRequestHandler(
+                $app['event_command_bus'],
+                new CreateProductionValidator()
+            )
+        );
+
+        $app[AddEventRequestHandler::class] = $app->share(
```

Comment:
Maybe also mention "Production" in this class name (and `RemoveEventRequestHandler`)? For example:
```suggestion
        $app[AddEventToProductionRequestHandler::class] = $app->share(
```
It's already in the Productions namespace, but it may be clearer when looking at it without knowing the namespace + the other request handlers also have it in their name (e.g. `RenameProductionRequestHandler` instead of `RenameRequestHandler`)

Reply:
Just a nit though

Reply:
Good point, I also was doubting this so I made it consistent now.